### ISSUE TITLE
After logging in, Nothing happens

### ISSUE THREAD

**Issue by OctoLiam** [AlexCSDev/PatreonDownloader#203]
I run it, it opens chromium, I log into patreon without an issue then nothing happens

**Comment by jaytgagne** [comment]
I second that. Login with email and PW and then nothing happens. I have tried with extra command line options and without.  Also have tried it on Linux and Windows. Same results. Setting --log-save isn't creating a  log as far as I can tell. Anyone offer any advice on a fix?

**Comment by jaytgagne** [comment]
I'm realitively new to git so don't beat me up if I missed something or don't explain it well. I took the time to try and help out. 

I got it to work after grabbing things from a few different posts. It will require you to use the command line and compile it yourself but that's not too bad. Here is what I did. (Got it working on both Windows and Linux)

1. Get a copy of Skyl3lazer's version(the fork is older but it worked).  Either got to https://github.com/Skyl3lazer/PatreonDownloader and download his zip or if you have git installed(recommended) git clone https://github.com/Skyl3lazer/PatreonDownloader -b patch-1 --recursive
2. Unzip or clone and move (cd) to the  appropirate /Patreon/PatreonDownloader/ directory.
3. Compile with "dotnet build -c Release"
4. Move to (cd) /Patreon/PatreonDownloader/PatreonDownloader.App directory.
5. From here you should be able to run it. 
6. Use something like "dotnet run --url https://www.patreon.com/#ENTER SITE HERE#/posts --embeds --campaign-images --json --log-save --user-sub-directories --download-directory D:\  

Read the ReadMe file as well as and the Build instructions (https://github.com/AlexCSDev/PatreonDownloader/blob/master/docs/BUILDING.md) becauase you need the submodules.



That should do it.

**Comment by Liquidream** [comment]
Thx for suggesting a workaround @jaytgagne 
Alas, even after following these steps (correctly, as far as I can tell) - I'm still left with Patreon opening, me logging in (entering 2FA code), then it sitting on the "/home" page and nothing exported.
I tried manually sending the "Chrome automated" tab to the intended "/posts" URL, to see if that kicked things off - but still nothing happens.
(I've also confirmed that the code I built is definitely from Skyl3lazer's patch branch, with submodules obtained).
Weird.
